docs/source/cn/sbp.py

```diff
@@ -0,0 +1,34 @@
+import oneflow
+from docreset import reset_docstr
+
+reset_docstr(
+    oneflow.sbp.sbp,
+    r"""
+    一个 sbp 是一个表示全局视角下的 Tensor 与物理设备上的 Tensor 之间的映射关系的对象。
+    
+    sbp 有三种类型：
+    
+    1. :attr:`split` 表示物理设备上的 Tensor 是将全局视角下的 Tensor 切分得到的。切分时需要指定切分的维度。物理设备上的 Tensor 如果根据切分的维度被拼接，可以还原得到全局视角的 Tensor 。
```

Comment:
```suggestion
    1. split 
        表示物理设备上的 Tensor 是将全局视角下的 Tensor 切分得到的。切分时需要指定切分的维度。物理设备上的 Tensor 如果根据切分的维度被拼接，可以还原得到全局视角的 Tensor 。
```

感觉这里完全按照英文版的格式来做的话会更好，如果同意这么改的话请麻烦把2和3也这样改一下

Reply:
好的，我也觉得按照英文文档好一些